src/maintenance/repositories/custom.md

```diff
@@ -0,0 +1,24 @@
+# Custom Repositories
+
+Void supports user created repositories available either locally, or remotely.
```

Comment:
Maybe add some note here about packages not part of the main tree can seriously bork your system and there are very real security questions to answer when trusting a non-official repo?

Reply:
Yes, I had been trying to strongly imply that you would create these yourself. I will look into that.